bnr-utils/nz_azConnector/nz_azConnector.go

```diff
@@ -102,6 +102,7 @@ func parseArgs(conn *Conn, backupinfo *BackupInfo, othargs *OtherArgs) {
 
 func handleErrors(err error) {
     if err != nil {
+        fmt.Println(err)

```

Comment:
we should prepend error also if not already done .. 

> fmt.Printf("error: %s", err)

Reply:
Done